src/main/flight/pid.c

```diff
@@ -205,21 +209,18 @@ void pidController(const pidProfile_t *pidProfile, const rollAndPitchTrims_t *an
     static float previousRateError[2];
     static float previousSetpoint[3];
 
-    if(FLIGHT_MODE(HORIZON_MODE))
-        calcHorizonLevelStrength(pidProfile);
```

Comment:
It was here as you don't have to perform it per axis. Now you moved it to for loop it runs for pitch and roll